api/docfx.json

```diff
@@ -54,6 +54,10 @@
       "feedback_github_repo": "Microsoft/quantum",
       "feedback_product_url": "https://github.com/microsoft/quantum/issues"
     },
+    "titleSuffix": {
+      "*.md": "Q# reference - Microsoft Quantum"
+      }
```

Comment:
It looks like there is an extra `}` here that is preventing DocFX from provisioning the docset correctly.
```suggestion
```

Reply:
Thanks for the heads up. I'll fix and resubmit.